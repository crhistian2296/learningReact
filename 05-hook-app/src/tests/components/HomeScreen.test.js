import { shallow } from 'enzyme';
import HomeScreen from '../../09-UseContext/HomeScreen';

describe('Pruebas sobre <HomeScreen/>', () => {
  const wrapper = shallow(<HomeScreen />);
  test('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
