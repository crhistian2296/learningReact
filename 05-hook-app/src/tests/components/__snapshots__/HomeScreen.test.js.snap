// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre <HomeScreen/> should render correctly 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
</div>
`;
